nautobot/core/tests/test_utils.py

```diff
@@ -442,58 +442,58 @@ def test_get_filterset_parameter_form_field(self):
             location_fields = ["comments", "name", "contact_email", "physical_address", "shipping_address"]
             for field_name in location_fields:
                 form_field = filtering.get_filterset_parameter_form_field(dcim_models.Location, field_name)
-                self.assertIsInstance(form_field, django_forms.CharField)
+                self.assertIs(type(form_field), forms.MultiValueCharField)
```

Comment:
I changed all of these because `isinstance(field, ChoiceField)` was matching when the field was a `MultipleChoiceField` since it's a subclass of `ChoiceField`. Matching subclasses doesn't seem to be what we were trying to achieve here.

Reply:
Makes sense for the test to be more specific, agreed.